### PR TITLE
Added additional Mercator devices, SMA02P Ikuü Battery Motion Sensor,…

### PR DESCRIPTION
**Pull request by kiwizznz**
Added additional Mercator devices, SMA02P Ikuü Battery Motion Sensor, SMA03P Ikuü Temperature & Humidity Sensor, SMA04P Ikuü Battery Contact Sensor, SMCL01-ZB Ikuü Ikon Ceiling Light CCT, SMD4109W-RGB-ZB Ikuü Walter Downlight RGB + CCT, SPBS01G Ikuü Plug Base, SPP02G Ikuü Double Indoors Power Point (Updated SPP02GIP to read "Outdoors"), SSW01G Ikuü Single Switch, SSW02G Ikuü Double Switch, SSW03G Ikuü Triple Switch, SSW04G Ikuü Quad Switch."

Running all these devices on my zigbee2mqtt docker container on RPI without any (visible) issues.

FYI - First pull request/new to Git. Ran npm test and came across a duplicate warning:

` FAIL  test/index.test.js
  ● index.js › Verify definitions

    Duplicate fingerprint for SSW04G: {"modelID":"TS0014","manufacturerName":"_TZ3000_fxjdcikv"}

      277 |                     for (const foundFingerprint of foundFingerprints) {
      278 |                         if (equals(foundFingerprint, fingerprint)) {
    > 279 |                             throw new Error(Duplicate fingerprint for ${device.model}: ${JSON.stringify(fingerprint)});
          |                                   ^
      280 |                         }
      281 |                     }
      282 |

      at test/index.test.js:279:35
          at Array.forEach (<anonymous>)
      at Object.forEach (test/index.test.js:204:27)

Test Suites: 1 failed, 5 passed, 6 total
Tests:       1 failed, 173 passed, 174 total
Snapshots:   0 total
Time:        4.491 s
Ran all test suites matching /test/i.`

### CODE REVIEW

File: devices/mercator.js [hunk 1]
```diff
@@ -30,10 +31,108 @@ module.exports = [
         },
     },
     {
-        fingerprint: tuya.fingerprint('TS011F', ['_TZ3210_7jnk7l3k', '_TZ3210_raqjcxo5']),
```
Comment (Koenkk) [inline devices/mercator.js:33]: Why was `_TZ3210_raqjcxo5` removed?

Comment (kiwizznz) [reply devices/mercator.js:33]: @Koenkk 

Looks like I forgot to update when I was editing and left in _TZ3210_tfxwxklq - should be _TZ3210_raqjcxo5.

[{modelID: 'TS011F', manufacturerName: '_TZ3210_7jnk7l3k'}] is the outdoors double powerpoint, SPP02GIP
[{modelID: 'TS011F', manufacturerName: '_TZ3210_raqjcxo5'}], is the indoors double powerpoint, SPP02G

_TZ3210_tfxwxklq should be _TZ3210_raqjcxo5

Comment (Koenkk) [reply devices/mercator.js:33]: Can you update your PR?

Comment (kiwizznz) [reply devices/mercator.js:33]: Done - I think? It's showing a second commit.

**Comment by Koenkk** [comment]
Thanks, could you also make a pr to add all pictures for these devices here: https://github.com/Koenkk/zigbee2mqtt.io/tree/master/public/images/devices? file name should be `MODEL.jpg` in 150x150 dimensions.

**Pull request merged**